app/src/docker/docker-compose.yml

```diff
@@ -150,8 +123,8 @@ services:
     image: va/abd_vro-service-data-access:latest
     environment:
       RABBITMQ_PLACEHOLDERS_HOST: rabbitmq1
-      RABBITMQ_PLACEHOLDERS_USERNAME: guest
-      RABBITMQ_PLACEHOLDERS_USERPASSWORD: guest
+      RABBITMQ_PLACEHOLDERS_USERNAME: ${RABBITMQ_USERNAME}
+      RABBITMQ_PLACEHOLDERS_USERPASSWORD: ${RABBITMQ_PASSWORD}
```

Comment:
1. Have you considered how to have the local build working after making this change? There could be a lot of this kind of "environment variables".
2. I'll update LH* as well. But 1 needs to be addressed.

Reply:
The local build worked for me. It's because the variables are defined in the .env file